tests/Microsoft.Identity.Test.Integration.Win8/BrokerOnWin8Tests.cs

```diff
@@ -2,9 +2,6 @@
 // Licensed under the MIT License.
 
 using Microsoft.Identity.Client;
-#if !NET6_WIN
-using Microsoft.Identity.Client.Broker;
```

Comment:
I think this line should be left here right? Since it is the opposite?

Reply:
We will see if it compiles :)